v0/destinations/mp/transform.js

```diff
@@ -163,6 +163,10 @@ function getTransformedJSON(message, mappingJson, useNewMapping) {
     set(rawPayload, "$name", getFullName(message));
   }
 
+  if (!message.context.active && message.context.active === false) {
```

Comment:
Isn't this sufficient?

```suggestion
  if (message.context?.active === false) {
```

Reply:
Yes, it is sufficient. Both the previous conditions are actually doing the same thing.